docker/Dockerfile

```diff
@@ -34,7 +34,7 @@ ENV PATH=/root/miniconda3/bin:${PATH}
 
 # add conda channels and install softwares
 RUN conda config --add channels defaults && conda config --add channels conda-forge && \
-conda config --add channels bioconda && conda install -c dongqingsun maestro=1.1.0 && \
+conda config --add channels bioconda && conda install -c dongqingsun maestro=1.2.1 && \
```

Comment:
@DongqingSun We should point it to `liulab-dfci` channel instead of `dongqingsun` channel in this repository. You can keep this change to your own dongqingsun/MAESTRO repository.

Reply:
Sure, I will point it to `liulab-dfci` channel in the repository.